src/sentry/static/sentry/app/views/settings/account/accountIdentities.tsx

```diff
@@ -47,7 +56,7 @@ class AccountIdentities extends AsyncView {
   };
 
   renderBody() {
-    const isEmpty = this.state.identities.length === 0;
+    const isEmpty = this.state.identities?.length === 0;
```

Comment:
in case of `undefined` this will be `false` and I think we would like it to be `true`, so maybe we could do like:

```suggestion
    const isEmpty = !this.state.identities?.length;
```

Reply:
`identities` should only be `null` if there was an error, I think explicitly checking length is more correct, but they'll both behave the same because of AsyncView

Reply:
yes, it could only be null :)